Cython/Compiler/CmdLine.py

```diff
@@ -5,10 +5,32 @@
 from __future__ import absolute_import
 
 import os
+import copy
 from argparse import ArgumentParser, Action, SUPPRESS
 from . import Options
 
 
+def backup_Options():
```

Comment:
This is test code, it shouldn't live in `CmdLine.py`. Keep it somewhere in the corresponding `…/Tests/` package.

Reply:
@scoder  Introduced Utils.py in Cython/Compiler/Tests.